open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
<p>line 9-10, two empty lines</p><p> </p><p>should be 0 or 1</p><p> </p><p>line 16 add empty line</p><p> </p><p>line 15: remove To</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>